modules/engageyaBidAdapter.md

```diff
@@ -0,0 +1,68 @@
+# Overview
+
+```
+Module Name: Engageya's Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: reem@engageya.com
+```
+
+# Description
+
+Module that connects to Engageya's demand sources
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: 'test-div',
+            mediaTypes: {
+                banner: {
+                    sizes: [[300, 250]],  // a display size
+                }
+            },
+            bids: [
+                {
+                    bidder: "engageya",
+                    params: {
+						widgetId: '<put here widget id>',
```

Comment:
do you have some test values for widgetId and websiteId, that I can use to test if adapter is working? If I put some dummy values here, request to `recs.engageya.com/rec-api/getrecs.json` is blocked by CORS policy:

 
![image](https://user-images.githubusercontent.com/12955750/102246049-99ed1980-3efe-11eb-9f5d-615c69089a88.png)

Reply:
websiteId: 91140
widgetId: 85610

Please also surf from 127.0.0.1 instead of "localhost".

Reply:
still fails for me, even with `127.0.0.1` (in this case header is not present at all). When I use some other domain (like `jita.local.net`), it's missing port (:9999 in this case), so I assume that's why it fails.

Reply:
how do you test it? because I used hello_world example and everything works perfectly using 127.0.0.1:9999

Reply:
![Screenshot from 2020-12-18 17-34-48](https://user-images.githubusercontent.com/12955750/102638314-97393100-4157-11eb-9cb3-6c50520187eb.png)

Reply:
There was a weird issue that I've fixed now. Please try using localhost :)